lib/atomic_cms/engine.rb

```diff
@@ -4,10 +4,16 @@ class Engine < ::Rails::Engine
     isolate_namespace AtomicCms
 
     config.generators do |g|
-      g.test_framework      :rspec,        fixture: false
+      g.test_framework :rspec, fixture: false
       g.fixture_replacement :factory_girl, dir: 'spec/factories'
       g.assets false
       g.helper false
     end
+
+    initializer 'atomic_cms.action_controller' do |_app|
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.